docs/getting_started_windows_vulkan.md

```diff
@@ -0,0 +1,133 @@
+# Getting Started on Windows with Vulkan
+
+[Vulkan](https://www.khronos.org/vulkan/) is a new generation graphics and
+compute API that provides high-efficiency, cross-platform access to modern GPUs
+used in a wide variety of devices from PCs and consoles to mobile phones and
+embedded platforms.
+
+IREE includes a Vulkan/[SPIR-V](https://www.khronos.org/registry/spir-v/) HAL
+backend designed for executing advanced ML models in a deeply pipelined and
+tightly integrated fashion on accelerators like GPUs.
+
+This guide will walk you through using IREE's compiler and runtime Vulkan
+components.
+
+## Prerequisites
+
+You should already have IREE cloned and building on your Windows machine. See
+the [Getting Started on Windows with CMake](getting_started_windows_cmake.md)
+guide for instructions.
+
+You must have a physical GPU with drivers supporting Vulkan. We support using
+[SwiftShader](https://swiftshader.googlesource.com/SwiftShader/) (a high
+performance CPU-based implementation of Vulkan) on platforms where the
+[Vulkan-ExtensionLayer](https://github.com/KhronosGroup/Vulkan-ExtensionLayer)
+project builds, but it does not currently
+[build on Windows](https://github.com/KhronosGroup/Vulkan-ExtensionLayer/issues/16).
+
+Vulkan API version > 1.2 is recommended, though older versions with support for
```

Comment:
... recommended on desktop, ...

Reply:
"recommended where available" :P if you can get it on Android or other platforms great (wishful future thinking?)

(Also addressed the other comments)